docs/editors/overview.md

```diff
@@ -403,6 +402,130 @@ Language Server Protocol (LSP). These extensions are not necessary for Metals to
 function but they improve the user experience. To learn more about Metals
 extensions, see [integrating a new editor](new-editor.md).
 
+
+## Additional file types
+
+Not all features are supported in all possible scenarios, espacially when it comes to non-standard Scala files like Ammonite scripts, worksheets or sbt scripts.
+
+<table>
+<thead>
+  <tr>
+    <td />
+    <td align="center">Sbt scripts</td>
+    <td align="center">Worksheets</td>
+    <td align="center">Ammonite scripts</td>
+    <td align="center">Standalone Scala files</td>
+  </tr>
+</thead>
+<tbody>
+   <tr>
+    <td>Diagnostics</td>
+    <td align="center">Parsing errors</td>
```

Comment:
Do you think it'd be a good idea to mark these with a `*` and then at the bottom of the table include an explanation of what `parsing errors` means here? I think it may not be super clear for people what exactly this means.

Reply:
Changed to a tick with `*` with an explanation below.